pkg/cmd/ide.go

```diff
@@ -40,6 +41,15 @@ var ideCmd = &cobra.Command{
 			}
 		}
 
+		isChosenIdeInstalled := false
+		if chosenIde.Name == "VS Code" {
+			isChosenIdeInstalled = ide_util.CheckAndAlertVSCodeInstalled()
+
+			if !isChosenIdeInstalled {
```

Comment:
No need to return here, an info message is enough because if the user chooses to install the IDE, they would have to go back to daytona and choose vscode again.

Reply:
That was done to stop **saving** and **printing** the 'default ide as Vscode'

```
c.DefaultIdeId = chosenIde.Id

		err = c.Save()
		if err != nil {
			log.Fatal(err)
		}
content := fmt.Sprintf("%s %s", views.GetPropertyKey("Default IDE: "), chosenIde.Name)
```

Reply:
I understand. I'm saying that we should go through with that.

Say that I don't have `code` installed.

I select VS Code in the IDE list, we say to the user that they don't have `code` installed and don't save VS Code as the default. I then install `code` and am forced to run `daytona ide` again.
That's not ideal. I suggest that we only warn the user about not having code installed and that we do save VS Code as the default in this scenario.

Reply:
alright, will revert, 
But even after seeing the message 'default ide: vscode', I believe a user will still go back and retry the <ide> command to confirm if that vscode uninstall error is gone , hehe atleast users like me..

Reply:
done, Reverted.

Reply:
> But even after seeing the message 'default ide: vscode' and then installing, I believe a user will still go back and retry the command to confirm if that vscode uninstall error is gone , hehe atleast users like me..

And we this solution we covered both cases 😄 
Thanks for the revert.